src/utils/web3/wagmi.js

```diff
@@ -1,54 +1,33 @@
 import { supportedChains } from "./networks";
-import { configureChains, createClient, createStorage } from "wagmi";
-import { jsonRpcProvider } from "wagmi/providers/jsonRpc";
+import { createClient, createStorage } from "wagmi";
 import { MetaMaskConnector } from "wagmi/connectors/metaMask";
 import { FlintWalletConnector } from "./connectors/flint";
-import { CardanoWSCConnector } from "./connectors/cardano-wsc";
+import { getDefaultConfig, MilkomedaNetworkName } from "milkomeda-wsc-ui-test-beta";
 
 const CHAIN_ID = Number(process.env.REACT_APP_CHAIN_ID);
 
 export const supportedChain =
   supportedChains.find((chain) => chain.id === CHAIN_ID) ?? supportedChains[0];
 
-const { provider, webSocketProvider } = configureChains(
-  [supportedChain],
-  [
-    jsonRpcProvider({
-      rpc: () => ({
-        http: process.env.REACT_APP_BLOCKCHAIN_URI ?? ""
-      })
-    })
-  ]
-);
-
 export const metamaskConnector = new MetaMaskConnector({
   chains: [supportedChain]
 });
 
 export const flintWalletConnector = new FlintWalletConnector({
   chains: [supportedChain]
 });
-export const flintWSCConnector = new CardanoWSCConnector({
-  chains: [supportedChain],
-  options: {
-    name: "flint"
-  }
-});
-
-export const eternlWSCConnector = new CardanoWSCConnector({
-  chains: [supportedChain],
-  options: {
-    name: "eternl"
-  }
-});
-
-export const client = createClient({
-  autoConnect: true,
-  storage: createStorage({
-    storage: window.localStorage,
-    key: "djed.stablecoin"
-  }),
 
-  provider,
-  webSocketProvider
-});
+export const client = createClient(
+  getDefaultConfig({
+    oracleUrl: process.env.REACT_APP_WSC_ORACLE_URL,
+    blockfrostId: process.env.REACT_APP_WSC_BLOCKFROST_ID,
+    chains: [supportedChain],
+    network:
+      CHAIN_ID === 2001 ? MilkomedaNetworkName.C1Mainnet : MilkomedaNetworkName.C1Devnet,
```

Comment:
Remember that this web dashboard will be used for deployments of Djed on other EVM blockchains as well in the future. Therefore, we should not assume that, if the Chain_Id is different from 2001, we will be in C1Devnet.